test/Microsoft.TestPlatform.CrossPlatEngine.UnitTests/Client/Parallel/ParallelOperationManagerTests.cs

```diff
@@ -9,6 +9,7 @@
 using Microsoft.VisualStudio.TestPlatform.CrossPlatEngine.Client;
 using Microsoft.VisualStudio.TestPlatform.CrossPlatEngine.Client.Parallel;
 using FluentAssertions;
+using System.Threading.Tasks;
```

Comment:
I expect some analyzer to fail here because the using is not in the right location.

Reply:
Fixed.